media_driver/linux/common/ddi/media_libva_util.cpp

```diff
@@ -471,10 +471,16 @@ VAStatus DdiMediaUtil_AllocateSurface(
         gmmCustomParams.BaseAlignment = 4096;
         gmmCustomParams.NoOfPlanes    = mediaSurface->pSurfDesc->uiPlanes;
         gmmCustomParams.CpTag         = cpTag;
+
+        int32_t deviceId = mediaDrvCtx->iDeviceId;
+
         switch (tileformat)
         {
             case I915_TILING_Y:
-                gmmCustomParams.Flags.Info.TiledY = true;
+		if (deviceId > 0x5600 && deviceId < 0x56B4)
```

Comment:
Please refer http://inlubt0232.iind.intel.com:8080/source/xref/CELADON/vendor/intel/external/onevpl-intel-gpu/_studio/shared/include/mfxstructures-int.h#497 for dGPU device id.

Thanks,
Tianmi

Reply:
The id I got is from kernel/include/drm/i915_pciids.h#696